app/views/devise/sessions/_form.html.haml

```diff
@@ -1,24 +1,32 @@
-.mui-container-fluid
-    .mui-row
-        .mui-col-lg-8.mui-col-lg-offset-2.mui-col-md-12.modal
-            %span.logo-venue
-            %h4 Log in with Email address
-            = tag :div, id: dom_id(resource, :errors)
-            = form_with(model: resource, as: resource_name, url: session_path(resource_name), remote: true, class: 'mui-form') do |f|
-                .mui-textfield.mui-textfield--float-label
-                    = f.email_field :email, autofocus: true
-                    = f.label :email, 'Email address'
-                .mui-textfield.mui-textfield--float-label
-                    = f.password_field :password
-                    = f.label :password
-                - if devise_mapping.rememberable?
-                    = f.check_box :remember_me
-                    = f.label :remember_me
-                %div
-                .form-group
-                    = f.submit 'Login', class: 'form-submit'
-                    = button_tag 'Cancel', class: 'form-submit', onclick: 'cancelModal(event);' 
-            %div{style: 'margin-top: 20px;'} 
-            %h3 Log in using your social media account 
-            = link_to icon('fab', 'facebook-f', "Login with Facebook"), user_facebook_omniauth_authorize_path, remote: true, class: 'form-submit', onclick: "mui.overlay('on')"
-            = link_to icon('fab', 'google-plus-g', "Login with Google"), user_google_oauth2_omniauth_authorize_path, remote: true, class: 'form-submit', onclick: "mui.overlay('on')"
+%body.modal-open
+    .modal-backdrop.fade.in
+        .modal-dialog{role: "document"}
+            .modal-content
+                .modal-body
+                    .mui-container-fluid
+                        .mui-row
+                            .mui-col-lg-8.mui-col-lg-offset-2.mui-col-md-12.modal
+                                %span.logo-venue
+                                %br/
```

Comment:
I’m not a great fan of using br. Can you set a top and bottom margin on the h4 below?